src/style.css

```
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Avenir", Helvetica, sans-serif;
```

Шрифти краще всього задавати в `html` тезі
```suggestion
  font-family: "Avenir Heavy", Helvetica, sans-serif;
```